config/install.yaml

```diff
@@ -12718,8 +12718,8 @@ spec:
           value: kyverno-svc
         - name: TUF_ROOT
           value: /.sigstore
-        image: ghcr.io/kyverno/kyverno:latest
-        imagePullPolicy: Always
+        image: vyankatesh/kyverno:1.8
```

Comment:
Why this change ? , looks like added by mistake.

Reply:
yes, Fixed.